package.json

```diff
@@ -70,7 +70,6 @@
     "@web3-onboard/coinbase": "^2.4.1",
     "@web3-onboard/core": "^2.21.4",
     "@web3-onboard/injected-wallets": "^2.11.2",
-    "@web3-onboard/keystone": "^2.3.7",
```

Comment:
Looks like we won't be migrating to AppKit until after Q1 2025, so we'll eventually have to find a way to restore this package.